gitignore

```diff
@@ -6,6 +6,7 @@
 .envrc
 .idea
 .vscode
+.yarn
```

Comment:
Do we want to match the ignored files as documented here https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored

Reply:
Ah yes, thank you!